app/routes/events/components/EventEditor/EventEditor.css

```diff
@@ -5,11 +5,6 @@
   color: var(--lego-font-color);
 }
 
-.formField {
-  margin: 1px;
-  margin-left: 3px;
-}
```

Comment:
As far as I can see this is still in use - do you either want to keep it, or remove all the references to it from app/routes/events/components/EventEditor/index.tsx?

Reply:
Yes, that's a mistake. I found the fields to look better without this, but forgot to clean it up. I'll just keep it for now and note it for later.